include/effects/matrix/PatternSMSupernova.h

```diff
@@ -112,6 +108,8 @@ class PatternSMSupernova : public LEDStripEffect
         if (debris_item._state == 0 || x < 0 || x >= MATRIX_WIDTH ||
```

Comment:
Good opportunity to reflow this line!

Reply:
I actually didn't touch this in past because I'm not sure how to restructure it in a way to make it better. It's a whole bunch of terms no matter how you slice and dice it.  
I'm happy to take on a concrete proposal though! :)

Reply:
Had another look and came up with an idea after all. Thanks for the tickle! :)